CodeWarsExercises/Codewars/CountingSheepExercise.cs

```diff
@@ -0,0 +1,42 @@
+﻿namespace CodeWarsExercises.Codewars {
+    internal class CountingSheepExercise : IExercise {
+        public string Name => "Counting sheep...";
+
+        public static int CountSheeps(bool?[] places) {
+            if (places is null)
+                throw new ArgumentNullException(nameof(places));
+
+            int counter = 0;
+
+            foreach (var sheep in places) {
+                if (!sheep.HasValue)
+                    throw new ArgumentException("There's musn't be nulls for sheeps.");
+
+                if (sheep.Value)
+                    ++counter;
```

Comment:
hope you remember the difference between ++i and i ++ (but does not matter for this particular case

Reply:
Yes, I remember the difference between `++i` and `i++`. 

`++i` -- first of all add 1 to i and then return the value.
`i++` -- first of all return the value and then add 1 to i.

I hope I'm right from terminology point of view.